util/options_helper.h

```diff
@@ -625,6 +625,9 @@ static std::unordered_map<std::string, OptionTypeInfo>
         {"index_per_partition",
          {offsetof(struct BlockBasedTableOptions, index_per_partition),
           OptionType::kUInt64T, OptionVerificationType::kNormal, false, 0}},
+        {"partition_filters",
```

Comment:
Does partitioned filters always comes with partitioned index? In that case can we have one less options, and just enable partition filter if index_type==kTwolevelIndexSearch and filter_policy != null?